docs/en/core-services/Caching.md

```diff
@@ -321,8 +321,12 @@ To enforce the assumption that tiers are configured from fastest to slowest, All
 blocks around tiers based on block annotation policies.
 
 Below configuration is honoured by each individual tier management task:
-- `alluxio.worker.management.task.thread.count`: How many threads to use for management tasks. (Default: `CPU core count`)
-- `alluxio.worker.management.block.transfer.concurrency.limit`: How many block transfers can execute concurrently. (Default:`CPU core count`/ 2)
+- `alluxio.worker.management.task.thread.count`: How many threads to use for management tasks. 
+  (Default: the maximum value of `4` and `CPU core count`.
+  Check the up-to-date [reference]({{ 'en/reference/Properties-List.html#alluxio.worker.management.task.thread.count' | relativize_url }}).)
```

Comment:
i think the relative links need the leading `/`?
```suggestion
  Check the up-to-date [reference]({{ '/en/reference/Properties-List.html#alluxio.worker.management.task.thread.count' | relativize_url }}).)
```

Reply:
good catch, thank you @Xenorith ! just fixed it, could you take a look again?

Reply:
the build caught it too
```
2022/04/19 20:49:48 Errors found in documentation markdown
  /tmp/docsCheck902138475/docs/en/core-services/Caching.md:
    326: relative link pointed to unknown file en/reference/Properties-List.html
    329: relative link pointed to unknown file en/reference/Properties-List.html
```
but i wrote the check so i can catch easier. now it looks good